src/importers/firefoxCsvImporter.ts

```diff
@@ -12,7 +12,9 @@ export class FirefoxCsvImporter extends BaseImporter implements Importer {
             return Promise.resolve(result);
         }
 
-        results.forEach(value => {
+        results.filter(value => {
+			return value.url !== "chrome://FirefoxAccounts"
```

Comment:
formatting (this line and L17 below it), alignment and spacing of indentation should match rest of document.